CHANGES.md

```diff
@@ -237,6 +240,10 @@ To be released.
     rather than the genesis block when there is a branch point.  [[#991]]
  -  `BlockPolicy<T>` became to validate `Block<T>.StateRootHash` property
      of a `Block<T>`.  [[#986]]
+ -  `Swarm<T>` became not to sync `Block<T>`s from the peers with
+    different genesis block.  [[#1003], [#1004]]
+ -  `Swarm<T>` became to ignore `BlockHeaderMessage` from the peers with
+    different genesis block.  [[#1003], [#1004]]
```

Comment:
Have these bugs existed before #1003? Aren't they regressions made from #1003?

Reply:
Yes there was. I've added `SwarmTest.Preload.PreloadIgnorePeerWithDifferentGenesisBlock()` and `SwarmTest.BroadcastIgnoreFromDifferentGenesisHash()` test for the case.